docs/build/html/_sources/app.rst

```diff
@@ -0,0 +1,109 @@
+App Module
+==========
+
+The `App` Module contains two files in it, `image_processor.py` and `input_manager.py`. When the application starts,
```

Comment:
in the initial paragraph you say "its these two files" this is nit picky but id change the wording to not make it seem like you are talking to an audience. maybe something like "these two files initiate..."